pylon/src/nginx.conf.template

```diff
@@ -33,20 +33,35 @@ http {
   #       can be simply set to 127.0.0.1.
   resolver 127.0.0.1;
 
+  # Turn gzip on
+  map $request_uri $routing_accept_encoding {
+    default "";
+    ~*/static "gzip";
+  }
+
   server {
     listen      80;
     server_name localhost;
+    client_max_body_size 256M;
+    client_body_buffer_size 256M;
+    proxy_connect_timeout 60m;
+    proxy_send_timeout 60m;
+    proxy_read_timeout 60m;
+    send_timeout 60m;
 
+    #
     # General-purpose reverse proxy.
-    location ~ ^/r/(http|https)/([^/]+)/(\d*)(.*)$ {
-      set $target_scheme $1;
-      set $target_host   $2;
-      set $target_port   :$3;
-      set $target_path   $4$is_args$args;
+    # The proxy to WebHDFS API depends on it.
+    #
+
+    location ~ ^/a/([^/]+)/(\d*)(.*)$ {
+      set $target_host   $1;
+      set $target_port   :$2;
+      set $target_path   $3$is_args$args;
       if ($target_port = ":0") {
         set $target_port "";
       }
-      proxy_pass $target_scheme://$target_host$target_port$target_path;
+      proxy_pass http://$target_host$target_port$target_path;
```

Comment:
>http [](start = 17, length = 4)

Will not support https by now?

Reply:
If the communication inside the cluster does not have to enable SSL, then we don't have to support https.